SparkPost/2web2ui
# Inbox Placement Landing Page / List Tests - SA-755

This is based off #1125 and #1130 

How to Test:

Start inbox placement locally and point to it on openresty
In current codebase, the endpoint is /api/v1/inbox-placement
Unless you've working project, you can modify controller.listCampaigns (in resources/controller.js) to return a list of tests (src/pages/inboxPlacement/tests/mockTests.js is a good test list to use)
Set the account option (not UI option), inbox_placement to be true.
Start the app and you should see Inbox Placement on Nav
Turn off local inbox-placement api to see error page
Return empty list of tests in local api to see empty state page
